spec/models/search/tag_query_spec.rb

```diff
@@ -158,13 +160,12 @@
   end
 
   before do
-    @wrangled_tag = create(:freeform, name: "wrangled ff")
-    create(:common_tagging, common_tag_id: @wrangled_tag.id)
+    create(:common_tagging, common_tag_id: wrangled_tag.id)
     update_and_refresh_indexes('tag', 1)
```

Comment:
`update_and_refresh_indexes` is only necessary if we're updating the shard count, for most tests it's overkill. But I can move things around later in [AO3-5581](https://otwarchive.atlassian.net/browse/AO3-5581).

Reply:
Fixed in #3512.